qa/workunits/rados/test_librados_build.sh

```diff
@@ -56,6 +73,7 @@ function run_binaries () {
 }
 
 pushd $DESTDIR
+install g++ make libradospp-dev librados-dev
```

Comment:
On SUSE (and - I believe - also in RHEL/CentOS), the RPM containing `/usr/bin/g++` is called "gcc-c++", not "g++". The others should be OK.

Reply:
ack. will fix this in my next revision!